osu.Game/Skinning/LegacySkin.cs

```diff
@@ -389,7 +397,45 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
 
                     return null;
 
+                case GameplaySkinComponent<GameplaySkinSamples> sampleComponent:
+                    var applause = GetSample(new SampleInfo("applause"));
+
+                    switch (sampleComponent.Component)
+                    {
+                        case GameplaySkinSamples.Applause:
+                            if (applause != null)
+                                return new DrawableSample(applause);
+
+                            break;
```

Comment:
This can also be handled using the existing `SampleInfo` fallback:

```csharp
new SampleInfo("applause", "Results/applause-d")
```

With the changes to the skin lookup, this should prefer the default if the lazer default is selected, but use classic `applause` if any classic skin is applied.